docs/administration-guide/data-permissions.md

```diff
@@ -0,0 +1,93 @@
+# Data permissions
+
+This page covers permissions for databases and tables. If you haven't already, check out our [Permissions overview][permissons-overview].
+
+## Permissions view
+
+Now that you have some groups, you’ll want to control their data access by going to the **Permissions** section of the Admin Panel. You’ll see an interactive table that displays all of your databases and all of your groups, and the level of access your groups have for each database.
+
+![Permissions view](images/permissions.png)
+
+You can click on any cell in the table to change a group’s access level. When you’re done making your changes, just click the `save changes` button in the top-right, and you’ll see a confirmation dialog summarizing the changes.
+
+### Unrestricted access
+
+Members of the group can access data from all tables (within all namespaces/schemas, if your database uses those), including any tables that might get added to this database in the future.
+
+### Granular access
+
+__Granular access__ allows administrators to explicitly select tables or schemas within a database. In practice, this means that:
+
+- If a new table gets added to this database in the future, the group won't get access to that new table. An administrator would need to explicitly grant access to that table.
+- Saved questions based on tables the group doesn’t have access to won’t show up in the list of saved questions.
```

Comment:
> Saved questions based on tables the group doesn’t have access to won’t show up in the list of saved questions.

That's not true — if you set a DB to Granular access, tables are either "Unrestricted," "No self-service," or "Sandboxed," and all of those levels would allow you to see questions based on that table if they're saved in a collection you have access to.